addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/DenonMarantzState.java

```diff
@@ -58,6 +58,10 @@ public DenonMarantzState(DenonMarantzStateChangedListener listener) {
         this.listener = listener;
     }
 
+    public void connectionError(String errorMessage) {
+        listener.connectionError(errorMessage);
```

Comment:
What if the `listener` is `null`?

Reply:
It can't be. State is instantiated from the `DenonMarantzHandler` like this, the handler is the (only) listener.
```java
denonMarantzState = new DenonMarantzState(this);
```

Reply:
Yes I saw that is is initialized with the handler, but code gets changed over time and who knows if there won't be another instance created somewhere in the future? Just saying this because with the term `listener` I usually connect that it is a 0-n relationship, i.e. it is optional or there may be many.

Reply:
I agree, will rename the variable to `handler` to avoid future confusion.